lineapy/transformer/linea_transformer.py

```diff
@@ -0,0 +1,171 @@
+                    # in that case, we pick the actual attribute.
+                    arg = argument_values[0][3]
+                    argnode = self.tracer.executor._value_to_node[arg]
+                else:
+                    argnode = self.tracer.lookup_node(arg, None).id
+                # TODO - might not need to look up the latest mutated node here. will revisit
+                argnode_latest = (
+                    self.tracer.mutation_tracker.get_latest_mutate_node(
+                        argnode
+                    )
+                )
+                dependent = [argnode_latest]
+                args_to_pass = [
+                    self.tracer.executor.get_value(argnode_latest),
+                    argument_values[1],
+                ]
+                to_exec = _save  # type: ignore
+            if func[1] == "get":
+                artifact_name = argument_values[0]
+                args_to_pass = [artifact_name]
+                to_exec = _get  # type: ignore
+
+            if func[1] == "artifact_store":
+                to_exec = _artifact_store  # type:ignore
+
+            lineanode = LineaCallNode(
+                id=get_new_id(),
+                session_id=self.tracer.get_session_id(),
+                # function_id=get_new_id(),
+                artifact_name=artifact_name,
+                artifact_version=None,
+                positional_args=[
+                    PositionalArgument(
+                        id=d,
+                        starred=False,
+                    )
+                    for d in dependent
+                ],
+                keyword_args=[],
+                module_name=func[0],
+                function_name=func[1],
+            )
+
+            # TODO - save the artifact here and attach the result to the graph.
+            print(
+                "I'm gonna do it.. i'm gonna save the artifact here. ready or not"
+            )
+            print(artifact_name, args_to_pass)
+            args_to_pass = [
+                lineanode,
+                self.tracer.executor,
+                self.tracer.db,
+            ] + args_to_pass
+            print(to_exec)
+            # set the equivalent of
+            try:
+                value = to_exec(*args_to_pass)
+                # the tracer dict linea_node_id_to_value preserves linea nodes across runs
+                self.tracer.linea_node_id_to_value[lineanode.id] = value
+                # the executor dict is the one that are used during execution. the tracer
+                # dict containing linea node values gets copied over pre-populating the executor
+                self.tracer.executor._id_to_value[lineanode.id] = value
+            except ArtifactSaveException as exc_info:
+                logger.error("Artifact could not be saved.")
+                logger.debug(exc_info)
+                return node
+
+            return lineanode
+
+        return node
+
+    def lvisit_Attribute(self, node: ast.Attribute):
```

Comment:
As I mentioned in slack, I worry this doesn't cover the case of `from lineapy import save, get`.